python/cdp/test/test_e2e.py

```diff
@@ -196,7 +196,7 @@ async def test_send_wait_and_get_user_operation_with_smart_account(cdp_client):
 @pytest.mark.asyncio
 async def test_send_transaction(cdp_client):
     """Test sending a transaction."""
-    account = await cdp_client.evm.get_account(name="E2EServerAccount")
+    account = await cdp_client.evm.create_account()
```

Comment:
test failing otherwise. maybe some account-scoped issue for "E2EServerAccount"

Reply:
I'm worried that we'll hit faucet rate limits again, but this should be fine to unblock for now